devsetup/Makefile

```diff
@@ -48,6 +48,7 @@ EDPM_NETWORKER_SUFFIX ?= 0
 EDPM_COMPUTE_ADDITIONAL_NETWORKS ?= '[]'
 EDPM_TOTAL_NODES ?= 1
 EDPM_COMPUTE_CEPH_ENABLED ?= true
+EDPM_COMPUTE_CEPH_NOVA ?= true
```

Comment:
we likely should not be defaulting to deploying with ceph in general
so can we set the default for both of these to false?

Reply:
we can do, in a different patch, I didn't plan to change deafult deployment modes